collapsiblecalendarview2/src/main/res/values/dimens.xml

```diff
@@ -6,8 +6,8 @@
     <dimen name="title_bar_button_height">36dp</dimen>
 
     <!-- day -->
-    <dimen name="day_circle_width">24dp</dimen>
-    <dimen name="day_circle_height">24dp</dimen>
+    <dimen name="day_circle_width">24sp</dimen>
+    <dimen name="day_circle_height">24sp</dimen>
```

Comment:
By converting from **dp** to **sp**, will this impact accessibility when adjusting display size?

Reply:
Display size will impact both `dp` and `sp`. As I tested, by converting from `dp` and `sp`, the display looks all good when use large display + large font size, large display + small font size, small display + large font size, and small display + small font size.